include/quic/gnutls_crypto.hpp

```diff
@@ -107,9 +107,9 @@ namespace oxen::quic
     inline constexpr size_t GNUTLS_SECRET_KEY_SIZE = 64;
 
     // These bytes mean "this is a raw Ed25519 private key" in ASN.1 (or something like that)
-    inline const std::string ASN_ED25519_SEED_PREFIX = oxenc::from_hex("302e020100300506032b657004220420"sv);
+    inline const std::string_view ASN_ED25519_SEED_PREFIX{oxenc::from_hex("302e020100300506032b657004220420"sv)};
     // These bytes mean "this is a raw Ed25519 public key" in ASN.1 (or something like that)
-    inline const std::string ASN_ED25519_PUBKEY_PREFIX = oxenc::from_hex("302a300506032b6570032100"sv);
+    inline const std::string_view ASN_ED25519_PUBKEY_PREFIX{oxenc::from_hex("302a300506032b6570032100"sv)};
```

Comment:
This has to remain `std::string`: the string_view will reference the string, but the string gets destroyed immediately after the statement.

Reply:
These*

Reply:
you're so right